src/polymer/element-descriptor.ts

```diff
@@ -197,8 +197,10 @@ function _getFlattenedAndResolvedBehaviors(
     const behavior = document.getOnlyAtId('behavior', behaviorName);
     if (!behavior) {
       throw new Error(
-          `Unable to resolve behavior \`${behaviorName}\` ` +
-          `Did you import it? Is it annotated with @polymerBehavior?`);
+          `In ${document &&
```

Comment:
Recommend reverting this for now, @FredKSchott is working on a more proper Warning based error here.